scripts/docker/mxe-build-container/instructions.md

```diff
@@ -5,10 +5,10 @@ This document assumes you have alreay installed docker and have checked out subs
 If you just want to build with the current mxe build container then starting from the folder above subsurface run
 
 ```bash
-docker run -v $PWD/win32:/win/win32 -v $PWD/subsurface:/win/subsurface --name=mybuilder -w /win -d subsurface/mxe-build-container:x.y /bin/sleep 60m
+docker run -v $PWD/win32:/win/win32 -v $PWD/subsurface:/win/subsurface --name=mybuilder -w /win -d subsurface/mxe-build:x.y /bin/sleep 60m
 ```
 
-replacing the x.y in the mxe-build-container tag with the current version e.g. 2.0
+replacing the x.y in the mxe-build tag with the current version e.g. 2.0
```

Comment:
I'm being silly - but since there isn't a `mxe-build:2.0` maybe we should list a valid version (`3.1.0`) here.

This is a typical case of a reviewer desperately trying to provide value, even if there really isn't any :)

Reply:
Good point, I have updated the example.